examples/datasets.yml

```diff
@@ -6,6 +6,11 @@ data:
     files:
       - nyc_taxi_wide.parq
 
+  - url: https://s3.amazonaws.com/datashader-data/nyc_taxi.zip
+    title: 'NYC Taxi Data'
+    files:
+      - nyc_taxi.csv
+
```

Comment:
We'll probably want to eliminate the .csv version at some point, as it is slower and isn't meant to have any data not in nyc_taxi_wide.parq, but for now it's fine to have both.

BTW, can you please search for nyc_taxi_50k.parq in the tutorials/ ; I think at some point we tell people to use that manually if they don't have the big dataset, and they'll have to be pointed to the actual location of this file instead (add ".data_stubs/", which is nice and obvious that it's temporary).

Reply:
Oh I see that. I am adding it back in.